modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -668,6 +668,84 @@ Total number of requests that uploaded an object to cloud storage.
 
 ---
 
+[[tls_metrics]]
+== TLS metrics
+
+=== redpanda_tls_truststore_expires_at_timestamp_seconds
+
+The expiration time, represented as a unix-style timestamp, of the shortest-lived CA in the installed CA chain. This lists all resources that have at least one CA configured.
```

Comment:
```suggestion
The expiration time, represented as a Unix-style timestamp, of the shortest-lived CA in the installed CA chain. This lists all resources that have at least one CA configured.
```
Any chance we could also get the full term for "CA" (unless we've already done that elsewhere in this doc)?